apps/src/aiTutor/views/UserMessage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import {
+  AITutorInteractionStatus as Status,
+  ChatCompletionMessage,
+} from '@cdo/apps/aiTutor/types';
+
+import style from './chat-workspace.module.scss';
+
+interface UserMessageProps {
+  message: ChatCompletionMessage;
+}
+
+const INAPPROPRIATE_MESSAGE = 'This chat is inappropriate.';
+const TOO_PERSONAL_MESSAGE = 'This chat is too personal.';
+const ERROR_MESSAGE =
+  'There was an error getting a response. Please try again.';
+
+const statusToStyleMap = {
+  [Status.OK]: style.userMessage,
+  [Status.UNKNOWN]: style.userMessage,
+  [Status.PROFANITY_VIOLATION]: style.inappropriateMessage,
+  [Status.PII_VIOLATION]: style.tooPersonalMessage,
+  [Status.ERROR]: style.errorMessage,
+};
+
+const getMessageText = (status: string, chatMessageText: string) => {
```

Comment:
I added `statusToStyleMap` and `getMessageText` helpers, but nothing else has changed. One of my next tasks is ensuring flagging the user messages appropriately works, so this will get tested again.